base/linalg/bidiag.jl

```diff
@@ -38,7 +38,7 @@ Bidiagonal(dv::AbstractVector, ev::AbstractVector) = throw(ArgumentError("did yo
 Constructs an upper (`uplo='U'`) or lower (`uplo='L'`) bidiagonal matrix using the
 given diagonal (`dv`) and off-diagonal (`ev`) vectors.  The result is of type `Bidiagonal`
 and provides efficient specialized linear solvers, but may be converted into a regular
-matrix with [`full`](:func:`full`). `ev`'s length must be one less than the length of `dv`.
+matrix with [`Array(_)`](:func:`convert`). `ev`'s length must be one less than the length of `dv`.
```

Comment:
`convert`? Would `...may be converted into a regular matrix with [`convert(Array, _)`](:func:`convert`) (or `Array(_)` for short)...` be better? Thoughts? Thanks!